### ISSUE TITLE
Unable to Install application on my localhost, any manual installation guide??

### ISSUE THREAD

**Issue by starpunjabi** [crater-invoice/crater#928]
For installation, i downloaded latest release i.e. https://github.com/crater-invoice/crater/archive/refs/tags/6.0.6.zip

Unzipped and Tried to run it by using
`php artisan serve`

It shows installation wizard in browser, but i got stuck in database setup slide, as it is showing errors like
![image](https://user-images.githubusercontent.com/20169304/168492687-03252a75-e35e-4105-afde-e9bb97581997.png)

Is there any way to skip installation wizard steps and install application manually?? As i tried on three machines, macbook air, macbook pro and windows 11. On all machines i got stuck at this stage, so there must be some issue in application (might be in latest release).

=======
**In order to solve this, Steps I tried are:**

I imported db tables by using 
`php artisan migrate`

and see database too by using
`php artisan db:seed`

Created a file under storage/app directory
![image](https://user-images.githubusercontent.com/20169304/168492896-c32a326f-9694-468b-9ba6-96a6aee9dff8.png)

and was able to skip installation steps, now it is showing login screen, but when i am trying to login I am getting error like
![image](https://user-images.githubusercontent.com/20169304/168493013-84daf0ac-e144-491f-944c-d192da76518a.png)

As per console error is with http request /api/v1/bootstrap
![image](https://user-images.githubusercontent.com/20169304/168493067-9952d560-6363-4a12-b3c6-ef71d09c922b.png)

Getting response like
`{"message":"Unauthenticated."}`
![image](https://user-images.githubusercontent.com/20169304/168493087-6a11bc40-752a-4552-97d4-16a39b259b72.png)

Can someone please let me know what is wrong with application. How to get it resolved.

Very many thanks!!

**Comment by starpunjabi** [comment]
Please check this video for installation steps: https://www.youtube.com/watch?v=waheY7LHBdc

**Comment by lMagickl** [comment]
I guess you didn't modified theses variables :
SANCTUM_STATEFUL_DOMAINS=127.0.0.1:8000
SESSION_DOMAIN=127.0.0.1
Because it work on my localhost and why I empty thses two varibles like this :
SANCTUM_STATEFUL_DOMAINS=
SESSION_DOMAIN=
I get the same error as you
![image](https://user-images.githubusercontent.com/33726502/169668553-f8cc450c-8893-4116-a9ea-3fe0b2ab6f22.png)

**Comment by rihards-simanovics** [comment]
Hi @lMagickl, you shouldn't primarily rely on the browser console to solve the issue. You need to check PHP logs and the logs of the engine you are using, Apache or Nginx. Also, check laravel logs and post them here. Go through the basics, check the file permission, start a new and seed the DB. Also, SESSION_DOMAIN is needed if you want to have any login functionality.

Also not sure if that is still the case, but you may want to try to download the crater invoice from the official site instead of GitHub. I remember I had some issues with the Github version.

**Comment by lMagickl** [comment]
My bad I didn't notice that @starpunjabi answered to himself x) I tried to help him

**Comment by 89itworld** [comment]
Yes it works for me, Your video is really awesome. Thanks a lot @starpunjabi
